browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
```

Comment:
Why don't you use `import`? Please unify the coding style, `require` or `import`